AuxTel/Standard-Operations/Nighttime-Operations/Monitoring-LATISS-Disperser-Wheel-Position.rst

```diff
@@ -0,0 +1,144 @@
+
+.. Include one Primary Author and list of Contributors (comma separated) between the asterisks (*):
+.. |author| replace:: *Kris Mortensen*
+.. If there are no contributors, write "none" between the asterisks. Do not remove the substitution.
+.. |contributors| replace:: *Erik Dennihy*
+
+.. This is the label that can be used as for cross referencing this procedure.
+.. Recommended format is "Directory Name"-"Title Name"  -- Spaces should be replaced by hyphens.
+.. _Monitoring-LATISS-Disperser-Wheel-Position:
+.. Each section should includes a label for cross referencing to a given area.
+.. Recommended format for all labels is "Title Name"-"Section Name" -- Spaces should be replaced by hyphens.
+.. To reference a label that isn't associated with an reST object such as a title or figure, you must include the link an explicit title using the syntax :ref:`link text <label-name>`.
+.. An error will alert you of identical labels during the build process.
+
+##########################################
+Monitoring LATISS Disperser Wheel Position
+##########################################
+
+.. _Monitoring-LATISS-Disperser-Wheel-Position-Overview:
+
+Overview
+========
+
+.. This section should provide a brief, top-level description of the procedure's purpose and utilization. Consider including the expected user and when the procedure will be performed.
+
+When night observing with AuxTel, it is important to check that the grating linear stage in properly aligned 
+to allow proper dispersion of the diffraction orders across the CCD. This check is especially critical if the 
+ATSpectrograph cRIO has been reset recently as the reset may result in mechanism timeouts or miscommunication. 
+During normal operations, the position of the disperser wheel can be monitored with telemetry from Chronograph or LOVE. 
+However, when the cRIO is reset and the linear stage is in the forward position, the reported position is no longer reliable. 
+Therefore, it is important to validate the position of the linear grating stage independently from the telemetry.
+
+.. _Monitoring-LATISS-Disperser-Wheel-Position-Precondition:
+
+Precondition
+============
+
+.. This section should provide simple overview of preconditions before executing the procedure; for example, state of equipment, telescope or seeing conditions or notifications prior to execution.
+.. It is preferred to include them as a bulleted or enumerated list.
+.. If there is a different procedure that is critical before execution, carefully consider if it should be linked within this section or as part of the Procedure section below (or both).
+
+- The telescope is currently on-sky and running the scheduler.
+
+- The linear stage position telemetry displayed abnormal values some time within the past 24 hours.
+
+  - The typical allowed range of positions for the disperser wheel position is 67-140mm (hard limit at 65mm).
```

Comment:
The values quoted here are the distances from the forward flange, but the values in chronograf and in the screen shot above are the distance from the home or reference position of the linear stage. So, it's better to say 0-67mm to align with what people should expect to see in chronograf. 

Also, the hard limit is a little past 67.5mm (I don't know the exact position). But there is a software limit of 67.5mm. That is why we set it to 67mm (the most forward position we can reliably reach).